### PR TITLE
stolostron/backlog#20329--reorg the app topic, add placement

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: applications/app_console.adoc [hunk 1]
```diff
@@ -3,94 +3,36 @@
...
-* If you selected Argo applications, you can view your Argo applications and launch to the Argo editor.
-
-* View cluster CPU and memory from the cluster node, on the properties dialog. 
-+
-**Notes:** The cluster CPU and memory percentage that is displayed is the percentage that is currently utilized. This value is rounded down, so a very small value might display as `0`.
-
-+
-For Helm subscriptions, see xref:../applications/package_overrides.adoc#configuring-package-overrides[Configuring package overrides] to define the appropriate `packageName` and the `packageAlias` to get an accurate topology display.
-+
+* Click the *Advanced configuration* tab to view terminology and tables of resources for all applications. You can find resources and you can filter subscriptions, placement rules, and channels. If you have access, you can also click multiple **Actions**, such as Edit, Search, and Delete.
```
Comment (chenz4027) [inline applications/app_console.adoc]: You can find resources and you can filter subscriptions, placement rules, and channels.
---------
Placement is missing from the description

File: applications/app_console.adoc [hunk 2]
```diff
@@ -3,94 +3,36 @@
...
-+
-Click the name of the resource node or select *Actions* > *View application* to see the details about the Ansible task deployment, including Ansible Tower Job URL and template name. Additionally, you can see errors if your Ansible Tower deployment is not successful.
-
 * Click *Launch resource in Search* to search for related resources.
 
-Use _Search_ to find application resources by the component `kind` for each resource. To search for resources, use the following values:
+* Use _Search_ to find application resources by the component `kind` for each resource. To search for resources, use the following values:
 
++
 |===
```
Comment (chenz4027) [inline applications/app_console.adoc:36]: I think we are also missing placement here. Should be:

Application resource | Kind (search parameter)
-- | --
Placement | Placement

**Review by chenz4027 — CHANGES_REQUESTED** [review]

File: applications/app_console.adoc (hunk 2 above)
Comment (swopebe) [reply applications/app_console.adoc:36]: Thanks for catching those, change is in.

**Review by chenz4027 — APPROVED** [review]
lgtm, thanks @swopebe

**Pull request merged**